api-java-mixin.raml

```diff
@@ -260,6 +450,78 @@ types:
+      }
+  ProductSetCustomerAction:
+    (java-mixin): |
+      public static ProductSetCustomerAction ofUnset() {
+          return ProductSetCustomerAction.of();
+      }
+  ProductSetProductPriceCustomFieldAction:
+    (java-mixin): |
+      public static ProductSetProductPriceCustomFieldAction ofUnset(final String name, final String priceId) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).build();
+      }
+
+      public static ProductSetProductPriceCustomFieldAction ofUnsetStaged(final String name, final String priceId,
+              @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).staged(staged).build();
+      }
+  ProductSetAttributeAction:
+    (java-mixin): |
+      public static ProductSetAttributeAction ofVariantId(final Long variantId, final String name, final com.fasterxml.jackson.databind.JsonNode value, final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).name(name).value(value).staged(staged).build();
+      }
+
+      public static ProductSetAttributeAction ofSku(final String sku, final String name, final com.fasterxml.jackson.databind.JsonNode value, final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().sku(sku).name(name).value(value).staged(staged).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttribute(final Long variantId, final String name) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).name(name).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttribute(final Long variantId, final String name, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).name(name).staged(staged).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttributeForVariantId(final Long variantId, final String name) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).name(name).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttributeForSku(final String sku, final String name) {
+          return ProductSetAttributeActionBuilder.of().sku(sku).name(name).build();
+      }
+
+      public static ProductSetAttributeAction ofUnsetAttributeForSku(final String sku, final String name, final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().sku(sku).name(name).staged(staged).build();
+      }
+  ProductSetAttributeInAllVariantsAction:
+    (java-mixin): |
+      public static ProductSetAttributeInAllVariantsAction ofUnsetAttribute(final String name) {
+          return ofUnsetAttribute(name, null);
+      }
+
+      public static ProductSetAttributeInAllVariantsAction ofUnsetAttribute(final String name, @javax.annotation.Nullable final Boolean staged) {
```

Comment:
```suggestion
      public static ProductSetAttributeInAllVariantsAction ofUnsetAttribute(final String name, final Boolean staged) {
```

Reply:
I removed all of Nullable annotation from the api-java-mixin file